proposed/2022/dotnet-nuget-why-proposal.md

```diff
@@ -0,0 +1,140 @@
+# dotnet nuget why command
+
+- Status: **Draft**
+- Author: [Kartheek Penagamuri](https://github.com/kartheekp-ms)
+- GitHub Issue [11782](https://github.com/NuGet/Home/issues/11782)
+
+## Problem Background
+
+Developers are often confused with top-level & transitive dependencies in their project. A developer should be able understand where every package originated from. A user should be able to promote a transitive dependency quickly & easily to a top-level dependency in the case there is a security concern on the package version depended on or if promoting to a top-level package will resolve a conflict.
```

Comment:
This is being pedantic, but I'd remove this part: 

> Developers are often confused with top-level & transitive dependencies in their project.

Developers want to know why a package is installed, that's the motivation, the confusion part leaves a bad taste in my mouth personally.

Reply:
Agreed, I can change the wording.